src/columnresizing.js

```diff
@@ -72,9 +72,9 @@ function handleMouseMove(view, event, handleWidth, cellMinWidth, lastColumnResiz
     if (target) {
       let {left, right} = target.getBoundingClientRect()
       if (event.clientX - left <= handleWidth)
```

Comment:
Confirmed that the proposed fix indeed fixes the same issue I'm seeing within my project.

As an alternative suggestion, I have a fork that works around this as follows:
```
// some edge cases, the `clientX` is less than the bounding rect of the target element.
// For these cases, we should select the cell on the right
if (event.clientX < left)
  cell = edgeCell(view, event, "left")  
else if (event.clientX - left <= handleWidth)
  cell = edgeCell(view, event, "left")
else if (right - event.clientX <= handleWidth)
  cell = edgeCell(view, event, "right")
```